tool/gravity/cli/install.go

```diff
@@ -725,6 +726,9 @@ func join(env *localenv.LocalEnvironment, environ LocalEnvironmentFactory, confi
 
 // TerminationHandler implements the default interrupt handler for the installer service
 func TerminationHandler(interrupt *signals.InterruptHandler, printer utils.Printer) {
+	audit := auditlog.New()
+	audit.AddDefaultRules()   //nolint:errcheck
+	defer audit.RemoveRules() //nolint:errcheck
```

Comment:
This can potentially not execute if the installer terminates in a some kind of ungraceful way. So I'm curious, what is the downside (if any) of these rules not being removed if that happens?

Reply:
I haven't done exhaustive testing but batching and grouping fields and syscalls seems to be the recommended way to reduce kernel churn and limit the results.

Not sure what we can do else to counter the ungraceful termination other than explicitly considering or ignoring the rest of the signals that have termination as the default behavior and can be handled.